.travis.yml

```diff
@@ -35,24 +35,21 @@ jobs:
         - make lint
 
     - stage: "Test on amd64"
-      name: "build, smallbuild, crossbuild"
```

Comment:
Why delete the small build here?

Reply:
Now smallbuild is default.

Reply:
This should be done in another PR, not in this one.

Reply:
I think build and smallbuild are somthing duplicate, combine them is better.